src/components/common/YourCompany/YourCompany.vue

```diff
@@ -45,7 +45,7 @@
             </template>
 
             <!-- Name Request Info -->
-            <template v-if="hasNewNr && (isAlterationFiling || isFirmChangeFiling || isSpecialResolutionFiling)">
+            <template v-if="hasNewNr">
```

Comment:
For all filings, if there's a new NR then display this block. (The alternative was to add correction filings to the if statement, but the new logic is even simpler and still correct.)